### PR TITLE
Replace metrics page with Mathjax rendered HTML

### PR DESCRIPTION
**Pull request by dplarson**
This PR replaces the Metrics page on the website (which currently links to a Word doc and PDF copy of the metrics proposal document) with a HTML page that includes all the selected metrics, with their definitions rendered in LaTeX using the Mathjax plugin.

**NOTE:** The individual commits can (and probably should) be squashed into a single commit before merging in. But I'm leaving them unsquashed at the moment to make reviewing a little easier. When the PR is approved, I can squash the commits.

### CODE REVIEW

**Comment by dplarson** [comment]
Here's a screenshot showing the style of the Mathjax rendered page:

Screen Shot 2019-06-28 at 11 26 34 AM

**Comment by lboeman** [comment]
I've already sacrificed my githubpages so I pulled your changes in and rendered them here: https://lboeman.github.io/metrics/  for reviewing.

**Comment by cwhanse** [comment]
Rendering of equations looks fine to me. Is it easy to add an index, perhaps in the left column, like we did for the Use Cases?

**Comment by lboeman** [comment]
> Rendering of equations looks fine to me. Is it easy to add an index, perhaps in the left column, like we did for the Use Cases?

It should be pretty straightforward. I can make a PR to add it after this is merged or add commits to this PR if that's helpful.

**Comment by dplarson** [comment]
I think it would be better to merge this PR first and then work on adding an index afterwards.

Are there any issues, comments, etc. with the metrics page or is this ready to be merged?

Note: I did not include the references listed in the metrics proposal document and as Cliff's comments in issue #71, I tried to remove the "editorial" statements from the descriptions.

File: metrics.md [hunk 1]
```diff
@@ -1,9 +1,267 @@
...
+The OVER metric can be normalized using the same approach as for KSI.
+
+
+### Combined Performance Index (CPI)
+The CPI can be thought of as a combination of KSI, OVER, and RMSE:
+
+$$ \text{CPI} = \frac{1}{4} ( \text{KSI} + \text{OVER} + 2 \times \text{RMSE} ) $$
+
+
+## Metrics fo Deterministic Event Forecasts
```
Comment (wholmgren) [inline metrics.md]: typo

Comment (dplarson) [reply metrics.md]: Good catch. I just pushed a commit to correct this typo.

**Comment by dplarson** [comment]
@wholmgren  @lboeman should I squash the commits for this PR or can you do it when merging via Github? (I think Github supports squashing commits when merging, but I can't remember.)

File: metrics.md [hunk 2]
```diff
@@ -1,9 +1,267 @@
...
+
+$$ \text{MBE} = \frac{1}{n} \sum_{i=1}^n (F_i - O_i) $$
+
+
+### Root Mean Square Error (RMSE)
+The RMSE is the square root of the averaged of the squared differences between the forecasted and observed values, and is defined as:
+
+$$ \text{RMSE} = \sqrt{ \frac{1}{n} \sum_{i=1}^n (F_i - O_i)^2 } $$
+
+RMSE is a frequently used measured for evaluating forecast accuracy. It can be interprest
```
Comment (lboeman) [inline metrics.md]: One more minor thing at this line and this is ready to go.

**Comment by lboeman** [comment]
@dplarson We can squash it while mergine the PR, thanks. Sorry for the late catch on that last typo.

File: metrics.md (hunk 2 above)
Comment (dplarson) [reply metrics.md]: Fixed.

**Comment by dplarson** [comment]
@lboeman no worries. Thanks for catching that typo.

**Pull request merged**